### PR TITLE
Show Client running out of Credits in Admin overview

### PR DESCRIPTION
**Pull request by 1day2die**
Dont know if this is needed, but its kinda a "funny to know" piece of information and there was an Issue open for this matter

image

### CODE REVIEW

**Comment by anondev-sudo** [comment]
When user ran out of credits, should be displayed

File: app/Models/User.php [hunk 1]
```diff
@@ -242,6 +242,26 @@ public function creditUsage()
         return number_format($usage, 2, '.', '');
     }
 
+
+    /**
+    * @description Returns the Users "out of Credits" time
+    *
+    * @return string
+    */
+    public function outOfCredits()
```
Comment (AVMG20) [inline app/Models/User.php:251]: purchase phpstorm or install a beutify plugin :)

**Review by AVMG20 — CHANGES_REQUESTED** [review]

File: app/Models/User.php (hunk 1 above)
Comment (1day2die) [reply app/Models/User.php:251]: DocBlock or Formatting ?

File: app/Models/User.php [hunk 2]
```diff
@@ -242,6 +242,28 @@ public function creditUsage()
         return number_format($usage, 2, '.', '');
     }
 
+
+    /**
+    * @description Returns the Users "out of Credits" time
+    *
+    * @return string
+    */
+     * @return string
```
Comment (AVMG20) [inline app/Models/User.php:251]: what's going on here

**Review by AVMG20 — CHANGES_REQUESTED** [review]

**Comment by anondev-sudo** [comment]
I don't see it
image

**Pull request merged**